src/EditorFeatures/Core/GoToDefinition/AbstractGoToDefinitionService.cs

```diff
@@ -71,17 +71,16 @@ internal abstract class AbstractAsyncGoToDefinitionService : AbstractFindDefinit
 
             return await GoToDefinitionHelpers.GetDefinitionLocationAsync(
                 symbol,
-                document.Project.Solution,
+                project.Solution,
                 _threadingContext,
                 _streamingPresenter,
                 thirdPartyNavigationAllowed: isThirdPartyNavigationAllowed,
                 cancellationToken: cancellationToken).ConfigureAwait(false);
         }
 
         private async Task<INavigableLocation?> GetAlternativeLocationIfAlreadyOnDefinitionAsync(
-            Document document, int position, ISymbol symbol, CancellationToken cancellationToken)
+            Document document, Project project, int position, ISymbol symbol, CancellationToken cancellationToken)
```

Comment:
this def needs docs.  very strange to pass a Document and Project along.  What is the former used for at this point?

Reply:
Not much :)